http-gateway/swagger.yaml

```diff
@@ -465,6 +483,24 @@ paths:
           $ref: '#/components/responses/badRequest'
         401:
           $ref: '#/components/responses/unauthorized'
+    delete:
+      tags:
+        - 'Pending Commands'
+      summary: 'Cancel pending commands for a resource of a device'
+      description: |
+        Cancel all pending commands for a single resource of a device. To cancel one pending command set 'correlationIdFilter' query parameter of pending command.
```

Comment:
Pending command is identified by the `correlationId`. If not specified, all pending commands will be canceled.